src/include/kompute/Tensor.hpp

```diff
@@ -79,6 +96,13 @@ class Tensor
                  uint32_t elementTotalCount,
                  uint32_t elementMemorySize);
 
+    /**
+     * @brief Reserve memory on the tensor
+     *
+     * @param newSize the new size for reservation
```

Comment:
Docstrings need update as well

Reply:
```suggestion
     * @param elementTotalCount Total number of elements for new size
     * @param elementMemorySize Memory size of element 
```

It has to match the number of parameters, see suggestion